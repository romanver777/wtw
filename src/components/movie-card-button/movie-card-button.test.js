/* eslint-disable @typescript-eslint/no-unsafe-call */
/* eslint-disable @typescript-eslint/no-unsafe-member-access */
/* eslint-disable @typescript-eslint/no-unsafe-assignment */
import React from "react";
import TestRenderer from "react-test-renderer";
import MovieCardButton from "../movie-card-button/movie-card-button";

describe("MovieCardButton component", () => {
  const props = {
    xlink: "play",
    title: "Add",
    className: "btn--play",
    handleClick: () => ({}),
  };
  const Trenderer = TestRenderer.create(<MovieCardButton {...props} />);

  it("renders correctly", () => {
    expect(Trenderer.toJSON()).toMatchSnapshot();
  });
});
