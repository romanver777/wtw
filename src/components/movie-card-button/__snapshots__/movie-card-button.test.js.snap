// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCardButton component renders correctly 1`] = `
<button
  className="btn btn--play movie-card__button"
  onClick={[Function]}
  type="button"
>
  <svg
    height="20"
    viewBox="0 0 19 20"
    width="19"
  >
    <use
      xlinkHref="play"
    />
  </svg>
  <span>
    Add
  </span>
</button>
`;
